tools/topology/topology2/platform/intel/google-rtc-aec-reference.conf

```diff
@@ -89,10 +89,6 @@ Object.Base.route [
 		source	$GOOGLE_RTC_AEC_REF_SOURCE
 		sink    "google-rtc-aec.18.1"
 	}
-	{
-		source	"module-copier.18.2"
-		sink    "host-copier.27.capture"
-	}
```

Comment:
on main 29 is Echo reference and 27 is DMIC RTC AEC. So this doesnt work on "main"

Reply:
thanks.  Echo reference was updated to 29. updated